# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/fastfetch/t2-logo.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- fastfetch-2.8.6/src/logo/ascii/t2.txt.vanilla	2024-02-28 14:25:23.906298262 +0100
+++ fastfetch-2.8.6/src/logo/ascii/t2.txt	2024-02-28 14:26:31.232360393 +0100
@@ -1,6 +1,15 @@
-${c2}TTTTTTTTTT
-    tt   ${c1}222${c2}
-    tt  ${c1}2   2${c2}
-    tt     ${c1}2${c2}
-    tt    ${c1}2${c2}
-    tt  ${c1}22222${c2}
+
+                        //
+                       //
+ TTTTTTTTTTTTTTT      // ..::  :-:.
+       ttt           // .::.    .:-:
+       ttt          //  .:.       --
+       ttt         //            :-:
+       ttt        //           .:-:
+       ttt       //           :-:.
+       ttt      //          .--.
+       ttt     //         .:-:
+       ttt    //          ...
+       ttt   //
+            //         .:::::::::::.
+           //
